README.md

```diff
@@ -220,6 +221,7 @@ Looking for something to build? Check out [the suggestions list][suggestions].
 - [gleam_javascript](https://github.com/gleam-lang/javascript) - [📚](https://hexdocs.pm/gleam_javascript/) - Work with JavaScript types and values in Gleam
 - [javascript_dom_parser](https://github.com/lpil/javascript-dom-parser) - [📚](https://hexdocs.pm/javascript_dom_parser/) - Bindings to the JavaScript DOMParser API
 - [plinth](https://github.com/Crowdhailer/plinth) - [📚](https://hexdocs.pm/plinth/) - Bindings to Node.js and browser platform APIs
+- [weapp](https://github.com/kaiwu/weapp) - [📚](https://hexdocs.pm/weapp/) - gleam bindings for wechat miniprogram, a.k.a weapp
```

Comment:
This package runs on JS but I don't think it's about JS, so a different category would be best

Reply:
Thanks for clearing out, but it is about as much JavaScript as `plinth` does. See `plinth` provides bindings to node.js and browser environment so that applications can be written with gleam there. It's exactly the same scenario  as `weapp`  provides bindings to [wechat miniprogram](https://developers.weixin.qq.com/miniprogram/en/dev/framework/) so that applications can be written with gleam there too. That's how i can land the categories.

Just rebased the needed fix on upstream. 
Have a nice day

Reply:
Should I do another publish to hex.pm so that it aligns with `gleam run` ?